clientv3/balancer.go

```diff
@@ -47,6 +47,15 @@ type simpleBalancer struct {
 	// upc closes when upEps transitions from empty to non-zero or the balancer closes.
 	upc chan struct{}
 
+	// downc closes when grpc calls down() on pinAddr or the balancer closes.
```

Comment:
probably only close it when the connection does down; otherwise it's mixing up responsibilities